src/zenml/integrations/neural_prophet/__init__.py

```diff
@@ -0,0 +1,32 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.integrations.constants import NEURAL_PROPHET
+from zenml.integrations.integration import Integration
+from zenml.utils.source_utils import import_class_by_path
+
+
+class NeuralProphetIntegration(Integration):
+    """Definition of NeuralProphet integration for ZenML."""
+
+    NAME = NEURAL_PROPHET
+    REQUIREMENTS = ["neuralprophet>=0.3.2"]
```

Comment:
I'm guessing neuralprophet has a `torch` python dependency so the materializer works even when only installing this integration? If not we need to either 
- specify `torch` here as well
- install the `pytorch` integration in the example `setup.sh`/Readme
- long-term enable a way to specify cross-integration dependencies

Reply:
It does require torch actually :-)